src/UserService/appsettings.json

```diff
@@ -39,6 +39,7 @@
     "SearchUsersEndpoint": "UserService_SearchUsers",
     "CreateAdminEndpoint": "UserService_CreateAdmin",
     "FindParseEntitiesEndpoint": "UserService_FindParseEntities",
+    "CheckUserExistenceEndpoint": "UserService_CheckUserExistence",
```

Comment:
Maybe better users, we can check more than 1 user

Reply:
done